how-to/getting-started-with-r/getting_started_with_r.md

```diff
@@ -0,0 +1,72 @@
+![R logo](https://www.r-project.org/Rlogo.png)
+
+# Getting started with R
+
+R is a free, open-source high-level programming language and software environment used primarily for statistical computing and graphics. 
+
+## R at Healthify:
+R is currently used for web scraping and data processing tasks.
+
+## First steps:
+Start by downloading [R](http://cran.r-project.org/) and [RStudio](http://www.rstudio.com/ide/), the R IDE (optional, but highly recommended).
+
+## Getting help:
```

Comment:
Yes. Installing `R` will also provide the shell. From the terminal, the console is simply called with command `R`